test/Frontend/Inputs/TestPlugin.cpp

```diff
@@ -0,0 +1,50 @@
+//===---------------------- TestPlugin.cpp ----------------------*- C++ -*-===//
```

Comment:
Nit: the `-*- C++ -*-` markers are only needed for `.h` files, because there it's ambiguous whether the language is C or C++.